setup.py

```diff
@@ -56,6 +56,7 @@
     "pygments>=2.12.0,<3",
     "PyYAML>=5.4.0,<7",
     "requests",
+    "sqlite-vec==v0.0.1-alpha.10",
```

Comment:
How about we put this under `sqlite` extra?

Reply:
👍 put it under `sqlite_vec` extra